Assets/Scripts/Internal/WorldTime.cs

```diff
@@ -46,6 +46,12 @@ public DaggerfallDateTime Now
             get { return DaggerfallDateTime; }
         }
 
+        public bool isDayTime()
```

Comment:
WorldTime is just a value container. This doesn't belong here, I'm sorry. :) This functionality already exists in DaggerfallDateTime.IsDay. You can check from anywhere through the singleton like so:
    if (DaggerfallUnity.Instance.WorldTime.DaggerfallDateTime.IsDay) { }